camera/camera_manager.hh

```diff
@@ -11,26 +10,25 @@
 namespace jet {
 
 struct Calibration {
-    cv::Mat camera_matrix;
-    cv::Mat distortion_coefficients;
+  cv::Mat camera_matrix;
+  cv::Mat distortion_coefficients;
 };
+
 struct Camera {
-    std::string serial_number = "uninitialized";
-    std::string v4l_path;
-    int video_index = 0;
-    Calibration calibration;
+  std::string serial_number;
+  std::string v4l_path;
 };
 
 class CameraManager {
-    public:
-        CameraManager();
-        Camera get_camera(const std::string& serial_number) const;
-    private:
-        void load_configs();
-        std::optional<YAML::Node> read_YAML(const std::string& filepath) const;
-        std::optional<Camera> parse_config(const YAML::Node& cfg) const;
-        std::optional<int> follow_v4l_path(const std::string& path) const;
-        std::map<std::string, Camera> camera_map_;
+ public:
+  CameraManager();
+  Camera get_camera(const std::string& serial_number) const;
+  Calibration get_calibration(const std::string& serial_number) const;
+
+ private:
+  void load_configs();
+  std::map<std::string, Camera> camera_map_;
```

Comment:
I like naming things like this with a hint of the key type, eg `serial_to_camera_map_`